Cargo.toml

```diff
@@ -16,7 +16,7 @@ resolver = "2"
 
 # Patching unreleased crates
 [workspace.dependencies]
-tls_codec = { version = "0.3.0-pre.1", features = ["derive", "serde", "mls"] }
+tls_codec = { git = "https://github.com/RustCrypto/formats.git", rev="67ce6cc76ed2020c37db911036b058a2926f778b", features = ["derive", "serde", "mls"] }
```

Comment:
A reminder to change this before merging.

Reply:
I think it's not needed to block on this. I reverted to `0.0.3-pre.1` and opened #1334.